src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1450,17 +1450,20 @@ def aks_use_dev_spaces(cmd, client, name, resource_group_name, space_name='defau
     :param resource_group_name: Name of resource group. You can configure the default group. \
     Using 'az configure --defaults group=<name>'.
     :type resource_group_name: String
-    :param space_name: Name of the dev space to use.
+    :param space_name: Name of the new or existing dev space to select. Defaults to an interactive selection experience.
     :type space_name: String
-    :param parent_space_name: Name of a parent dev space to inherit from when creating a new dev space. \
-    By default, if there is already a single dev space with no parent, the new space inherits from this one.
-    :type parent_space_name: String
+    :param update: Update to the latest Azure Dev Spaces client components.
+    :type update: bool
+    :param prompt: Do not prompt for confirmation. Requires --space.
+    :type prompt: bool
     """
 
-    if _get_or_add_extension(DEV_SPACES_EXTENSION_NAME):
+    if _get_or_add_extension(DEV_SPACES_EXTENSION_NAME, update):
         azext_custom = _get_azext_module(DEV_SPACES_EXTENSION_NAME, DEV_SPACES_EXTENSION_MODULE)
         try:
-            azext_custom.ads_use_dev_spaces(name, resource_group_name, space_name, parent_space_name)
+            azext_custom.ads_use_dev_spaces(name, resource_group_name, space_name, update, prompt)
+        except TypeError:
+            raise CLIError("Use '--update' option to get the latest Azure Dev Spaces client components.")
```

Comment:
Currently, when we release a newer azure-cli, as we changing the signature here and that of the extension, if the user calls use command on the old extension, it will throw an error. The user would need to update the extension and then we should not see this error.

Reply:
Ok.